database.py

```diff
@@ -40,14 +56,21 @@ def __str__(self):
 
 
 class Database:
+    """The Database object is a container for all the supported Pokemon."""
     __pokemon_list = []
     __pokemon_dictionary = {}
+    __pokemon_type_dictionary = {}
+    __POKEMON_TYPES = ('normal', 'fire', 'fighting', 'water', 'flying', 'grass', 'poison',
+                       'electric', 'ground', 'psychic', 'rock', 'ice', 'bug', 'dragon', 'ghost',
+                       'dark', 'steel', 'fairy')
```

Comment:
__POKEMON_TYPES = ('normal fire fighting water flying grass poison electric ground psychic rock ice bug dragon ghost dark steel airy').split()  # easier to type, read, debug